lms/static/js/demographics_collection/DemographicsCollectionModal.jsx

```diff
@@ -269,15 +289,18 @@ class DemographicsCollectionModal extends React.Component {
           <Wizard.Page>
             {({ wizardConsumer }) =>
               <div className="demographics-form-container">
-                <label htmlFor={FIELD_NAMES.MILITARY}>Have you ever served on active duty in the U.S. Armed Forces, Reserves, or National Guard?</label>
+                {/* Military History */}
+                <label htmlFor={FIELD_NAMES.MILITARY}>
+                  {gettext("Have you ever served on active duty in the U.S. Armed Forces, Reserves, or National Guard?")}
+                </label>
                 <select
                   className="form-control"
                   onChange={this.handleSelectChange}
                   name={FIELD_NAMES.MILITARY}
                   id={FIELD_NAMES.MILITARY}
                   value={wizardConsumer[FIELD_NAMES.MILITARY]}
                 >
-                  <option value="default">Select service branch</option>
+                  <option value="default">{gettext("Select service branch")}</option>
```

Comment:
Should the default option match what we have in the account MFE? Our default option there is `Select military history`. We're not asking them to select the actual military branch they were in, so I feel the default option is a little misleading?

Reply:
I don't remember where I got this text from, but I defer to @ProductRyan on this.